src/components/Output.js

```diff
@@ -5,7 +5,7 @@ function Output(props) {
   return (
     <>
       <label className={props.wrapperclass}>
-        <div className="skew-background"></div>
+        <div className="skew-background" style={props.backgroundcolor}></div>
```

Comment:
I recommend finalizing converting CSS to styled-components first.
Now, you have to use inline-CSS to apply dynamic styling.

`style={props.backgroundcolor}` has some issues:
1) it should be `backgroundColor` 
2) I would expect that `backgroundColor` is a string like "#fff". This would result in `style={"#fff"}` which is invalid.